build.gradle

```diff
@@ -59,32 +59,32 @@ dependencies {
     implementation group: 'org.openjfx', name: 'javafx-swing', version: javaFxVersion, classifier: 'linux'
 
     //// JOGL
-    compile "org.jogamp.gluegen:gluegen-rt:2.3.1"
-    compile "org.jogamp.jogl:jogl-all:2.3.1"
-
-    runtime "org.jogamp.gluegen:gluegen-rt:2.3.1:natives-android-aarch64"
-    runtime "org.jogamp.gluegen:gluegen-rt:2.3.1:natives-android-armv6"
-    runtime "org.jogamp.gluegen:gluegen-rt:2.3.1:natives-linux-amd64"
-    runtime "org.jogamp.gluegen:gluegen-rt:2.3.1:natives-linux-armv6"
-    runtime "org.jogamp.gluegen:gluegen-rt:2.3.1:natives-linux-armv6hf"
-    runtime "org.jogamp.gluegen:gluegen-rt:2.3.1:natives-linux-i586"
-    runtime "org.jogamp.gluegen:gluegen-rt:2.3.1:natives-macosx-universal"
-    runtime "org.jogamp.gluegen:gluegen-rt:2.3.1:natives-solaris-amd64"
-    runtime "org.jogamp.gluegen:gluegen-rt:2.3.1:natives-solaris-i586"
-    runtime "org.jogamp.gluegen:gluegen-rt:2.3.1:natives-windows-amd64"
-    runtime "org.jogamp.gluegen:gluegen-rt:2.3.1:natives-windows-i586"
-
-    runtime "org.jogamp.jogl:jogl-all:2.3.1:natives-android-aarch64"
-    runtime "org.jogamp.jogl:jogl-all:2.3.1:natives-android-armv6"
-    runtime "org.jogamp.jogl:jogl-all:2.3.1:natives-linux-amd64"
-    runtime "org.jogamp.jogl:jogl-all:2.3.1:natives-linux-armv6"
-    runtime "org.jogamp.jogl:jogl-all:2.3.1:natives-linux-armv6hf"
-    runtime "org.jogamp.jogl:jogl-all:2.3.1:natives-linux-i586"
-    runtime "org.jogamp.jogl:jogl-all:2.3.1:natives-macosx-universal"
-    runtime "org.jogamp.jogl:jogl-all:2.3.1:natives-solaris-amd64"
-    runtime "org.jogamp.jogl:jogl-all:2.3.1:natives-solaris-i586"
-    runtime "org.jogamp.jogl:jogl-all:2.3.1:natives-windows-amd64"
-    runtime "org.jogamp.jogl:jogl-all:2.3.1:natives-windows-i586"
+    implementation "org.jogamp.gluegen:gluegen-rt:2.3.1"
+    implementation "org.jogamp.jogl:jogl-all:2.3.1"
+
+    runtimeOnly "org.jogamp.gluegen:gluegen-rt:2.3.1:natives-android-aarch64"
```

Comment:
My only concern which I'd need to test is that the platform specific binaries still need to be included in the jar. I imagine runtimeOnly is not included by default but I may be wrong.